src/Resources/contao/classes/Frontend.php

```diff
@@ -649,231 +649,19 @@ public static function indexPageIfApplicable(Response $objResponse)
 		}
 	}
 
-
 	/**
 	 * Check whether there is a cached version of the page and return a response object
 	 *
 	 * @return Response|null
+	 *
+	 * @deprecated This method will always return null and does not have any effect
+	 *             anymore. Caching is achieved using the powerful Symfony reverse
```

Comment:
If to use the Sf reverse proxy is a configuration of the application (`contao/standard-edition`), it could very well be Varnish or a CDN. Something like `Caching is handled by the reverse proxy, depending on your application setup`.